WalletWasabi.Fluent/Views/NavBar/NavBar.axaml

```diff
@@ -94,7 +94,11 @@
   </UserControl.Styles>
   <DockPanel Margin="0 0 0 4">
     <StackPanel DockPanel.Dock="Top" Spacing="5" Background="Transparent">
-      <Panel Height="30" />
+      <Panel Height="30" Background="#01FFFFFF">
+        <i:Interaction.Behaviors>
+          <behaviors:DragWindowBehavior/>
```

Comment:
Remove that behavior, setting removing Background="Transparent" from StackPanel and setting IsHitTestVisible="False" should be enough.